pkg/image/query.go

```diff
@@ -96,3 +97,55 @@ func FindByGalleryID(ctx context.Context, r Queryer, galleryID int, sortBy strin
 		},
 	}, &findFilter)
 }
+
+func FindGalleryCover(ctx context.Context, r Queryer, galleryID int) (*models.Image, error) {
+	const useCoverJpg = true
+	img, err := findGalleryCover(ctx, r, galleryID, useCoverJpg)
+	if err != nil {
+		return nil, err
+	}
+
+	if img != nil {
+		return img, nil
+	}
+
+	// return the first image in the gallery
+	return findGalleryCover(ctx, r, galleryID, !useCoverJpg)
+}
+
+func findGalleryCover(ctx context.Context, r Queryer, galleryID int, useCoverJpg bool) (*models.Image, error) {
+	// try to find cover.jpg in the gallery
+	perPage := 1
+	sortBy := "path"
+	sortDir := models.SortDirectionEnumAsc
+
+	findFilter := models.FindFilterType{
+		PerPage:   &perPage,
+		Sort:      &sortBy,
+		Direction: &sortDir,
+	}
+
+	imageFilter := &models.ImageFilterType{
+		Galleries: &models.MultiCriterionInput{
+			Value:    []string{strconv.Itoa(galleryID)},
+			Modifier: models.CriterionModifierIncludes,
+		},
+	}
+
+	if useCoverJpg {
+		imageFilter.Path = &models.StringCriterionInput{
+			Value: string(filepath.Separator) + "cover.jpg",
```

Comment:
Will droping the file separator and thus looking for a plain `cover.jpg` in the path get us a lot of false positives?
The existing behavior was having a suffix of `cover.jpg` so we are now requiring  the renaming of the image file to cover.jpg or making a copy of the file with that name.
Whatever is decided the help section -> Galleries needs to be updated

Reply:
Good pickup. Changed to be consistent with the old behaviour.